src/api/helpers/map-ordnance-survey-addresses/index.test.ts

```diff
@@ -0,0 +1,15 @@
+import mapOrdnanceSurveyAddresses from '.';
+import mapAddress from '../map-ordnance-survey-address';
+import mockOrdnanceSurveyResponse from '../../test-mocks/mock-ordnance-survey-response';
+
+const ordnanceSurveyResponse = mockOrdnanceSurveyResponse.results;
+
+describe('api/helpers/map-ordnance-survey-addresses', () => {
+  it('should return an array of mapped addresses', () => {
+    const result = mapOrdnanceSurveyAddresses(ordnanceSurveyResponse);
```

Comment:
Can this function ever return an empty array, if yes then an assertion for the same please.

Reply:
Yes, but we don't need to assert this. The implementation uses `.map` - the unit test mimics this.

If we were to add a test for an empty array, we would testing the native `.map` behaviour.

The test only needs to ensure that it's mapping over the addresses with `mapAddress`

Unit test coverage is reported at 100%.